script.js

```diff
@@ -40,48 +42,66 @@ SlidesElms[9].style.color= "yellow";
 BublesElms=document.querySelectorAll("h1");
 BublesElms[10].style.color= "tan";
 
-CabbinElms=document.querySelectorAll("h1");
-CabbinElms[11].style.color= "blue"
+CabbinElms= document.querySelectorAll("h1");
+CabbinElms[11].style.color= "blue";
 
-CarryElms=document.querySelectorAll("h1");
-CarryElms[12].style.color= "red"
+CarryElms= document.querySelectorAll("h1");
+CarryElms[12].style.color= "red";
 
 BakesElms=document.querySelectorAll("h1");
-BakesElms[13].style.color= "purple"
+BakesElms[13].style.color= "purple";
 
 
 
-let elms;
-elms = document.querySelectorAll("h1")
-console.log(elms)
+for (elms of elms);
+{
+  elms= style.backgroundColor("blue");
```

Comment:
The program is halting here because there is an error. `style` isn 't found. Be carful of the `=` sign.